pyproject.toml

```diff
@@ -66,20 +66,21 @@ dev-flake8 = [
 dev-formatting = ["black == 24.4.2", "isort == 5.12.0"]
 dev-mkdocs = [
   "black == 24.4.2",
-  "Markdown==3.4.4",
+  "Markdown==3.7",
   "mike == 2.0.0",
   "mkdocs-gen-files == 0.5.0",
   "mkdocs-literate-nav == 0.6.1",
   "mkdocs-macros-plugin == 1.0.4",
   "mkdocs-material == 9.4.4",
   "mkdocstrings[python] == 0.25.0",
+  "mkdocstrings-python == 1.9.2",
   "mkdocstrings-python == 1.10.8",
   "frequenz-repo-config[api] == 0.9.2",
```

Comment:
You should also bump the version here to `0.10.0` and for all occurrences of `frequenz-repo-config` in `pyproject.toml`.